src/controllers/authenticationController.ts

```diff
@@ -161,55 +163,47 @@ export default class AuthenticationController {
     };
 
     // Return the actual session information.
-    return right({
-      body: publicSession,
-      status: 200
-    });
+    return ResponseSuccessJson(publicSession);
   }
 
   /**
    * The Single logout service.
    */
-  public slo(): Either<Error, IResponse<string>> {
-    return right({
-      body: "/",
-      status: 301
-    });
+  public async slo(): Promise<IResponsePermanentRedirect> {
+    const url: UrlFromString = {
+      href: "/"
```

Comment:
@gunzip is this a valid `Url`?

Reply:
should use const errorOrUrl = UrlFromString.decode("/")